packages/cadl-rlc-test/test/parametrizedHost/cadl-output/src/parameters.ts

```diff
@@ -3,4 +3,14 @@
 
 import { RequestParameters } from "@azure-rest/core-client";
 
-export type ListCollectionsParameters = RequestParameters;
+export interface ListCollectionsQueryParamProperties {
+  /** The API version to use for this operation. */
+  "api-version": string;
```

Comment:
So because there is no default value so we generate the parameter for every operation, right?

Reply:
In this case, they have a apiVersion in @server, will need to check how cadl deal with it.